File: transpoze-matrix.py
# 867. Transpose Matrix

class Solution(object):
    def transpose(self, matrix):
        result = []
        for i in range(len(matrix[0])): 
            column = []
            for j in range(len(matrix)):  
                column.append(matrix[j][i])
            result.append(column)
        return result

File: transpoze-matrix.js
var transpose = function(matrix) {
  let result = [];
  for(let i = 0; i < matrix[0].length; i++){
      let column = [];
      for(let j = 0; j < matrix.length; j++){
          column.push(matrix[j][i]);
      };
      result.push(column);
  };
  return result
};